src/components/DataTable.vue

```diff
@@ -41,36 +41,30 @@ export default {
   },
   data() {
     return {
-      tableData: [],
+      postData: [],
       showDetail: false,
       postDetails: [],
     };
   },
 
   methods: {
     showRowDetail(tableItem) {
-      this.postDetails.push(tableItem)
-      console.log(this.postDetails)
-
+      this.postDetails.push(tableItem);
     },
 
-    deleteRow:function(data){
-      console.log(data)
-       this.tableData=this.tableData.filter((currentItem) => {
-         console.log(currentItem)
-         console.log(data)
-         console.log(currentItem.data.author)
-        
-        return data !== currentItem.data.id
-       });
+    deletePost(data) {
+      this.postData = this.postData.filter((currentItem) => {
```

Comment:
Redundant use of brackets. For a single argument in callback we can avoid using `()`